vignettes/stan-help.Rmd

```diff
@@ -0,0 +1,423 @@
+---
+title: Resources for help with Stan
+description: "How to address issues you may encounter with Stan"
+author: Michael DeWitt
+output: rmarkdown::html_document
+bibliography: library.bib
+csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+vignette: >
+  %\VignetteIndexEntry{Help with Stan}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = FALSE
+)
+```
+
+
+<!-->
+
+- [ ] What is Stan
+- [ ] Issues with installation
+- [ ] Issues with compilation and paths
+- [ ] Issues with model fitting
+  - [ ] Adapt Delta
+  - [ ] Max Tree Depth
+  - [ ] Number of samples, priors, Rhat/ ESS
+- [ ] Working with posteriors
+- [ ] Other resources
+
+<-->
+
+# Epinowcast and Stan
+
+[Stan](https://mc-stan.org/) is the probabilistic programming language and statistical platform for statistical modelling that powers the Bayesian inference in `{epinowcast}`.
+The statistical models used in `{epinowcast}` are primarily written in the [Stan programming language](https://mc-stan.org/docs/reference-manual/index.html), a statically typed programming language with syntax similar to C/C++.
+`{epinowcast}` utilises the [`{cmdstanr}`](https://mc-stan.org/cmdstanr/) package to interface with CmdStan, the program which executes the models written in the Stan programming language.
+It is important to understand that CmdStan is a program that is **distinct** from but interfaced through R.
+R calling a separate program to execute calculations is similar to [`{rjags}`](https://cran.r-project.org/web/packages/rjags/) which relies on the [JAGS library](https://cran.r-project.org/web/packages/rjags/INSTALL) and [R-INLA](https://www.r-inla.org/what-is-inla).
+As described in the `{epinowcast}` project [README](./index.html), you will need to install [`{cmdstanr}`](https://mc-stan.org/cmdstanr/) an R package which also has the ability to install CmdStan using an R interface. 
+Additionally, you will need to make sure that the software required by CmdStan is installed and configured on your machine.
+
+## Ensuring you have the proper toolchain {.toolchain}
```

Comment:
This is a nice addition to the install instructions we have in the README. we should make sure to clearly link to it from there.